src/components/FeatureViews/Accounts/TabComponents/StudentCourseViewer.js

```diff
@@ -172,26 +143,30 @@ const StudentCourseViewer = ({ studentID, current = true }) => {
 												<Moment
 													format="h:mm a"
 													date={
-														course.schedule.start_date +
-														course.schedule.start_time
+														enrollment.course.startDate
+														+ "T" +
+														enrollment.course.startTime
 													}
 												/>
 												{` - `}
 												<Moment
 													format="h:mm a"
 													date={
-														course.schedule.end_date + course.schedule.end_time
+														enrollment.course.endDate
+														+ "T" +
+														enrollment.course.endTime
```

Comment:
is there no better way to format this lol

Reply:
you gotta make it a datetime string somehow lol. you could make it a template literal i guess haha